src/view/drag-handle/sensor/create-keyboard-sensor.js

```diff
@@ -222,6 +223,11 @@ export default ({
       eventName: 'scroll',
       fn: callbacks.onWindowScroll,
     },
+    // Cancel on page visibility change
```

Comment:
I was able to simply the solution.

Reply:
We do not need to know what the visibility of the page is when the even fires - we simply need to know if the event fires. If it fires while a drag is occurring we can kill the drag